### DISCUSSION TITLE
Digital proof of academic status not accepted

### DISCUSSION THREAD

**Discussion by marleen01** [community/community#133833] [GitHub Education]
### Hi everyone,

I'm studying a fully distance learning program, and in my country's official digital government website, they have proof of academic status PDF, with QR codes to prove authenticity of the document. I used to be able to upload the PDF and it would work, but now it only asks me to use my camera. I tried taking a picture of the screenshot, and it didn't work. I also have no clue how to translate it as it seems to be a new requirement. For proof of being away from campus, in the degree it literally says distance learning, meaning no in-person classes. I'm unable to apply for Github Student anymore because of the new system.

**Comment by harisdev-netizen** [comment]
I understand your frustration with the new system for verifying academic status on GitHub Student. It sounds like you're facing a few challenges:

1. The system now only accepts camera-based verification instead of PDF uploads.
2. You're in a distance learning program, so you don't have traditional proof of being on campus.
3. The digital proof from your government website isn't being accepted.

Here are some suggestions to try and resolve this issue:

1. Contact GitHub Support: Reach out to GitHub's support team directly. Explain your situation in detail, mentioning that you're in a distance learning program and that the previous PDF upload method worked for you. Ask if there are alternative verification methods for students in online programs.
2. Try the mobile app: If you haven't already, try using the GitHub mobile app for verification. Sometimes the mobile verification process can be more flexible.
3. Check for updated documentation: Look for any updated documentation on GitHub's website about student verification for distance learning programs. They may have alternative procedures in place.
4. Use an official enrollment letter: Request an official enrollment letter from your university that specifically states you're enrolled in a distance learning program. Try uploading a photo of this document instead of the government PDF.
5. Seek help from your university: Contact your university's IT or student services department. They may be familiar with this issue and could have solutions or be able to provide alternative documentation that works with GitHub's system.
6. Look for workarounds: Some students have reported success by taking a photo of their student ID along with a handwritten note containing their GitHub username and the current date and I was one of them.

Remember, GitHub is generally supportive of students in various learning environments, including distance learning. It's likely that there's a solution available, even if it's not immediately apparent. Keep persisting and don't hesitate to explain your unique situation to their support team.

**Reply by samus-aran** [reply]
Great response, but wanted to correct this section below for yourself and everyone else.

> Contact GitHub Support: Reach out to GitHub's support team directly. Explain your situation in detail, mentioning that you're in a distance learning program and that the previous PDF upload method worked for you. Ask if there are alternative verification methods for students in online programs.

GitHub Education Support do not provide manual reviews. Please only reach out to Support if you encounter a bug, school details are incorrect and require to be updated, or have enquiries on Classroom and other GitHub Education partnerships.

**Reply by aseyedia** [reply]
Why did you think it was helpful to just copy and paste the ChatGPT response in here?

**Comment by samus-aran** [comment]
Hi @marleen01 

> I tried taking a picture of the screenshot, and it didn't work.

Can you explain why it didn't work?

Regards to translating your evidence, the application process does automatically translate most languages. Try it out first and see if it isn't to pick up your school name, etc. 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
